examples/CMakeLists.txt

```diff
@@ -133,4 +135,22 @@ install(TARGETS chol_driver DESTINATION "${CMAKE_INSTALL_PREFIX}")
 install(TARGETS schur_cg_driver DESTINATION "${CMAKE_INSTALL_PREFIX}")
 install(TARGETS hybrid_driver DESTINATION "${CMAKE_INSTALL_PREFIX}")
 install(TARGETS solver_driver DESTINATION "${CMAKE_INSTALL_PREFIX}")
-
+install(TARGETS solver_driver_emulator DESTINATION "${CMAKE_INSTALL_PREFIX}")
+
+
+if (HYKKT_USE_AMD)
+  if(NOT AMD_LIBRARY)
+    message(STATUS "Cannot find AMD, disabling SuiteSparse AMD module ...")
+    set(HYKKT_USE_AMD OFF CACHE BOOL "Build without SuiteSparse AMD module." FORCE)
+  else()
+    target_link_libraries(perm_driver PUBLIC AMD)
```

Comment:
We should either use a list here, or just have the AMD library be linked in through the HyKKT library itself